### ISSUE TITLE
Parameter Changes are not being saved in the Collection.

### ISSUE THREAD

**Issue by Siddhant125** [a85/POSTMan-Chrome-Extension#10319]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

When I change the parameter value for my request in Collection, it is not saving the changes after click on Save button.

Initial State
![image](https://user-images.githubusercontent.com/44401619/134390900-f44a0885-e065-4475-a783-c53a0920d132.png)

Changed the value, ran the request and clicked on Save.
![image](https://user-images.githubusercontent.com/44401619/134391077-9783a5cc-3421-4c41-bf1f-d6d3d0f93eae.png)

Refresh web page and again the old value is displayed.
![image](https://user-images.githubusercontent.com/44401619/134391252-7f3fadab-e3b4-43ad-a195-dfb96fc10776.png)

This collection is linked to an API and a monitor.
The same issue is observed in the desktop client as well. But there you just need to close the postman application instead of refreshing the web page.

### Steps To Reproduce

1. Open request tab.
2. Change Parameter value.
3. Run the request and save it.
4. Refresh the webpage.
5. Verify is the changed value is still reflected.

### Environment Information

```markdown
- Operating System: Windows 10 Version 21H1 Build 19043.1237
- Platform Type: Chrome Browser
- Postman Version: 8.12.2
```

**Comment by coditva** [comment]
@Siddhant125 I'm unable to replicate this issue. Can you share logs from Chrome DevTools Console with sensitive information removed or drop an email with the logs to help@postman.com mentioning this ticket ID?

**Comment by Siddhant125** [comment]
[Requests Changed.log](https://github.com/postmanlabs/postman-app-support/files/7243329/Requests.Changed.log)
[After Refresh.log](https://github.com/postmanlabs/postman-app-support/files/7243330/After.Refresh.log)

Here I have attached the 2 files - one when changes were made, saved and request was executed successfully. And the other after refresh where the changes were undone automatically.

**Comment by coditva** [comment]
@Siddhant125 Does this happen only with params or with auth/headers/scripts/body too? Are the changes made to auth/headers/scripts/body reflected after refresh?

**Comment by Siddhant125** [comment]
Yes, the issue is happening only with Params tab, all the other tabs work fine with save and refresh.

**Comment by coditva** [comment]
@Siddhant125 Got it. I would need a few more things:
1. Is this happening for only this request? Or all requests in this collection? Or all requests in Postman?
2. Can you send me the network logs? To find these:
	- open network tab in Chrome DevTools.
	- refresh the page.
	- perform replication steps.
	- click on the "Export HAR" button (the download icon) in header.
![image](https://user-images.githubusercontent.com/8445768/135087861-5e9e58ba-a9cd-47e8-814c-8d46cf187624.png)

The network logs might contain some sensitive information, so please send it to help@postman.com while mentioning this ticket ID.

**Comment by Siddhant125** [comment]
This is happening for 2 requests in the same collection, but they are under the same folder. There 2 more requests in the collection which are not under the same folder and there we don't observe this issue.

I have sent the HAR file on help@postman.com

**Comment by coditva** [comment]
@Siddhant125 Thanks for the quick responses and sending the logs. I've identified the issue in the logs. We'll be tracking this as a bug and will post any updates on this thread. 🙂

**Comment by Siddhant125** [comment]
Is there a workaround in the meantime while you are working on permanent fix for this bug?

**Comment by coditva** [comment]
@Siddhant125 I'm not 100% sure if this would work, but you can try duplicating the folder which has these requests.

![image](https://user-images.githubusercontent.com/8445768/135589630-75df5b78-c848-4613-9627-3ad243ad216a.png)

Let me know how this works out. 🙂
